### PR TITLE
add a confirmation modal to the file uploader adapter

### PR DESCRIPTION
**Pull request by felixsaintgenis**

### CODE REVIEW

**Comment by vincentfretin** [comment]
Si tu veux tu peut faire une fonction générique `confirmDeletionModal(callback)` et dans le `onClick`  `callback();`

et tu l'appelles :  `confirmDeletionModal(() => input.onChange(''))`

**Comment by MilanBarande** [comment]
I did that on the modify-slug branch (displayConfirmationModal)

File: assembl/static2/js/app/components/form/fileUploaderFieldAdapter.jsx [hunk 1]
```diff
@@ -13,6 +15,27 @@ type Props = {
...
+          closeModal();
+        }}
+        className="button-submit button-dark"
+      >
+        <Translate value="delete" />
+      </Button>
+    ];
+    return displayModal(confirmDeletionTitle, confirmDeletionBody, true, footer);
+  };
+
```
Comment (vincentfretin) [inline assembl/static2/js/app/components/form/fileUploaderFieldAdapter.jsx]: en fait tu peux réutiliser `confirmDeletionModal` de `fieldArrayWithActions.jsx`

`confirmDeletionModal(confirmDeletionTitle, confirmDeletionBody, () => input.onChange(''))`

Comment (felixsaintgenis) [reply assembl/static2/js/app/components/form/fileUploaderFieldAdapter.jsx]: oui effectivement

**Comment by vincentfretin** [comment]
Ok, good enough for me. Milan on your modify-slug, try to reuse the same function or modify it to be more generic if needed and move it in another file.

**Pull request merged**